doc/COMPILE.md

```diff
@@ -52,56 +54,65 @@ installation of self-compiled Elektra (such as how to uninstall it).
 > If they are not available, some of the functionality gets disabled automatically.
 > The core of Elektra never depends on other libraries.
 
-To build documentation you need doxygen (we recommend 1.8.8+), graphviz and [ronn-ng](https://github.com/apjanke/ronn-ng/blob/master/INSTALLING.md):
+### Documentation dependencies
 
-```sh
-apt-get install doxygen graphviz
-gem install ronn-ng -v 0.10.1.pre1
-```
+To build documentation you need doxygen (we recommend 1.8.8+), graphviz and [ronn-ng](https://github.com/apjanke/ronn-ng/blob/master/INSTALLING.md). These can be installed as follows:
```

Comment:
Useful changes and clarifications :+1:. It seems like a determiner slipped your attention here. 
```suggestion
To build the documentation you need doxygen (we recommend 1.8.8+), graphviz and [ronn-ng](https://github.com/apjanke/ronn-ng/blob/master/INSTALLING.md). These can be installed as follows:
```